scripts/tag_bioasq_test_data.py

```diff
@@ -0,0 +1,45 @@
+import xml.etree.ElementTree as ET
+import typer
+from tqdm import tqdm
+import json
+
+from grants_tagger.predict import predict_tags
+
+
+def get_mesh2code(mesh_metadata_path):
+    mesh_tree = ET.parse(mesh_metadata_path)
+
+    mesh2code = {}
+    for mesh in tqdm(mesh_tree.getroot()):
+        try:
+            # DescriptorUI e.g. M000616943
+            mesh_code = mesh[0].text
+            # DescriptorName e.g. Mucosal-Associated Invariant T Cells
+            mesh_name = mesh[1][0].text
+            
+            mesh2code[mesh_name] = mesh_code
+        except IndexError:
```

Comment:
Why would IndexError happen, considering the name is correct ? Would be good to investigate further how many errors happened (maybe print in a log?)

Reply:
This is copy from https://github.com/wellcometrust/grants_tagger/blob/main/grants_tagger/filter_mesh_tags.py

I can definitely add logger or we can also refactor the code in the other function. That said, scripts in theory should be independent, right? I think the way the code is structured its not meant to share code.